src/blockchain_ledger_entry_v2.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+package helium;
+
+message blockchain_ledger_entry_v2 {
+  // NOTE: might need to add owner:binary here for portability reasons,
+  // leaving that out for now to revisit later.
+  uint64 hnt_nonce = 1;
```

Comment:
Have these subtypes been stable across hst and hnt or should these be broken into subrecords, like:
```
ledger_entry_v2{
  hnt_entry hnt_entry = 1,
  hst_entry hst_entry = 2,
  …
}

hnt_entry{
  uint64 balance = 1,
  uint64 nonce = 2
}

hst_entry{
  uint64 balance = 1,
  uint64 nonce = 2
}
…
```

Reply:
Sure, I'll break these out. But we probably would keep these static, not sure how well a generic version translates to rocks storage. I doubt these would be updated _very_ frequently :slightly_smiling_face:

Reply:
I thought you were convinced it'd be better to have the static ledger fields instead of the generic "list of ledger entries" which adds ore complexity on update/insert

Reply:
> I thought you were convinced it'd be better to have the static ledger fields instead of the generic "list of ledger entries" which adds ore complexity on update/insert

Oh, I wanted to avoid having `blockchain_ledger_entries_v2`, this one would work relatively okay. By static I meant not having generic `token_type` in the `ledger_entry` record.